packages/oid4vci-holder/src/agent/OID4VCIHolder.ts

```diff
@@ -634,7 +634,7 @@ export class OID4VCIHolder implements IAgentPlugin {
       })
 
       const credential = {
-        id: issuanceOpt.credentialConfigurationId ?? issuanceOpt.id,
+        id: (issuanceOpt.credentialConfigurationId ?? issuanceOpt.id) as string | undefined, // cast due to error:  Type 'unknown' is not assignable to type 'string | undefined'.
```

Comment:
```suggestion
        id: (issuanceOpt.credentialConfigurationId ?? issuanceOpt.id as string | undefined) , // cast due to error:  Type 'unknown' is not assignable to type 'string | undefined'.
```

Reply:
Well the bigger question is why it fails. I know that the lower level libs have some changes. But that has been fixed in develop. The fact develop builds and this branch doesn't suggest either the PR is reverting something, not using the latest develop, or new code is introduced where this happens. Please investigate first before continuing @sanderPostma

Reply:
Ah okay, now I understand what you mean. Yes this was fixed in lates develop

Reply:
@nklomp 
Looks like build are passing now indeed